stores/governance/src/main/resources/db/store/postgresql/V0_1100_1__init.sql

```diff
@@ -177,4 +177,61 @@ CREATE TABLE constitution
 );
 
 CREATE INDEX idx_constitution_slot
-    ON constitution (slot);
\ No newline at end of file
+    ON constitution (slot);
+
+DROP TABLE if exists local_gov_action_proposal_status;
+CREATE TABLE local_gov_action_proposal_status
+(
+    gov_action_tx_hash varchar(64),
+    gov_action_index   int,
+    status             varchar(20),
+    epoch              int,
+    slot               bigint,
+    create_datetime    timestamp,
+    update_datetime    timestamp,
+    PRIMARY KEY (gov_action_tx_hash, gov_action_index, epoch)
+);
+
+DROP TABLE IF EXISTS local_committee;
+CREATE TABLE local_committee
+(
+    threshold          double precision,
+    epoch              int,
+    slot               bigint,
+    update_datetime    timestamp,
+    PRIMARY KEY (epoch)
+);
+
+DROP TABLE IF EXISTS local_treasury_withdrawal;
+CREATE TABLE local_treasury_withdrawal
```

Comment:
We need this data in reward calculation. Should we make it like instant_reward table ? (something like this in adapot branch) https://github.com/bloxbean/yaci-store/blob/3fc5c61a62a1db8be115779f5b967b120b9e3da7/aggregates/adapot/src/main/resources/db/store/postgresql/V0_1300_1__init.sql#L34